apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -53,6 +69,8 @@ describe("Client API", () => {
   });
 
   it("clears the client_id from user metadata, tries creating a client without access token, creates a client, fails if another client request is made, gets the newly created client, and deletes the client", () => {
+    cy.wait(2000); // Because sometimes we hit the shortened rate limit
```

Comment:
auth0 is who is rate limiting us right?

Reply:
No, it's a self-imposed rate-limit for testing. See the `rateLimiting.cy.ts` file.

Reply:
There's no way to change the rate limit on the fly, so it affects all cypress tests.

Reply:
I've seen the rate limiting tests, but where is the rate limit enforced?

Reply:
It's in the initialization of the service. It's an express middleware.

Reply:
The rate is much lower if the ENV === "test"

Reply:
For anyone reading this after the fact, we're going to test some options of splitting the rate-limiting stuff into its own isolated context (separate workflow, etc), so the low, test-specific rate limits won't affect other tests.